src/entry/vexflow-core-with-bravura.ts

```diff
@@ -0,0 +1,21 @@
+// [VexFlow](https://vexflow.com) - Copyright (c) Mohit Muthanna 2010.
+// MIT License
+//
+// Support Dynamic Importing of Music Fonts
+//
+// vexflow-core-with-bravura.ts is the entry point for vexflow-core-with-bravura.js.
+// This version includes only Bravura.
+// It also overrides the `Flow.setMusicFont(...)` function to be async,
+// loading other music fonts (e.g., Petaluma) on the fly.
+
+import { Vex } from '../';
+import { loadMusicFonts } from '../fonts/bundleBravura';
+import { loadTextFonts } from '../fonts/textfonts';
+
+loadMusicFonts();
```

Comment:
Simon & Rodrigo were interested in loading fonts dynamically. The way we implemented it over the summer (with magic path aliases) didn't work well when published as an NPM module.

The different bundles we create now each have a separate entry file (see vexflow/src/entry/*). For example, this file (vexflow-core-with-bravura.ts) will eventually be built into vexflow/build/vexflow-core-with-bravura.js. That file bundles only one font, and can dynamically load the other two fonts if requestedvia Flow.setMusicFont(...).